### ISSUE TITLE
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.

### ISSUE THREAD

**Issue by livioavalle** [Creator13/mapsnap#1]
Hi.
I created a new project with this command `mapsnap init myTestProject "44.9087,8.6088" "44.9072,8.6117" 10`
and then after moving in the myTestProject folder with cd i run `mapsnap run`.
But the command gave me a lot of error line with this `Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.`

This is the full `mapsnap snap` output
`
Bounding box (size: 84x1, area: 84):
 (245384,0) - - (245468,0)
 |                       |
 (245384,1) - - (245468,1)
Final image size: 21504x256px (5,5MP)
Downloading 84 tiles: [                         ]   0% (0/84)Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Unexpected response from OSM tile server: 400 BadRequest. Please report to the developers.
Done.
Downloaded 84 tiles (0,0kB) in 1,885s (average size 0,0kB, average time 0ms)
Saving image (this may take a while if you're snapping a large area)...
Saved snapshot as myTestProject 2022-01-11 13_40_56.png!
`

**Comment by Creator13** [comment]
Hello! I am unable to reproduce this bug. Would you mind sharing the contents of the `mapsnap.json` file in the project folder?

**Comment by livioavalle** [comment]
Hi, sorry for the issue, but i think that was a temporary error, beacuase later that day it worked fine. I think it can be closed

**Comment by Creator13** [comment]
Alright, thanks for letting me know! I have run into `400 Bad Request` issues before, and possible causes are that the server was simply overloaded, or too many requests had been sent from one place before that. Prompted by this issue, I went over the [OSM Tile Server Usage Policy](https://operations.osmfoundation.org/policies/tiles/) once more, and I found some ways in which mapsnap breaches this policy, so I will push a small update to fix this soon! Hopefully that will reduce the chances of running into `400 Bad Request` in the future.
